### ISSUE TITLE
Retool events/event.js and PopcornEditor.js to support in-page embedding (?)

### ISSUE THREAD

**Issue by bvibber** [mozilla/popcorn-editor#49]
It's possible not everyone wants to use the iframe embedding, and may prefer their own HTML scaffolding and running in the main JS context.

If so, we could still provide the same API around events/event.js on the inside, and PopcornEditor.js on the outside.

Not sure this is needed/wanted. Thoughts?

**Comment by jbraun27** [comment]
Yes, I agree. It's been a while since I have looked at it. It would simplify event handling I think. Embed.js also will need some attention.
